packages/nodes-base/credentials/Aws.credentials.ts

```diff
@@ -295,8 +295,12 @@ export class Aws implements ICredentialType {
 			endpoint = new URL(requestOptions.url);
 			if (service === 'sts') {
 				try {
-					endpoint.searchParams.set('Action', 'GetCallerIdentity');
```

Comment:
the exact same code exists below in the custom-url section as well. we should maybe extract this code into a single private function.